Makefile

```
@@ -1,4 +1,4 @@
-.PHONY: help clean% check% format% lint test pyright playwright% install% testrail% coverage release
+.PHONY: help clean% check% format% docs lint test pyright playwright% install% testrail% coverage release
```

Would this cover `docs-preview`?

```suggestion
.PHONY: help clean% check% format% docs% lint test pyright playwright% install% testrail% coverage release
```

---

yeah, good idea, thanks!